hack/sanitize-coverage.sh

```
@@ -0,0 +1,219 @@
+        has_coverage_files=true
+        sanitize_html_coverage "coverage.html" "coverage-sanitized.html"
+    fi
+    
+    if [[ -f "cover.out" ]]; then
+        has_coverage_files=true
+        sanitize_file "cover.out" "cover-sanitized.out"
+    fi
+    
+    # Check for any other coverage files
+    local other_coverage_files
+    other_coverage_files=$(find . -path './vendor' -prune -o -path './vendor/bin/bundle/catalog' -prune -o -name "*.cover.out" -o -name "*.coverprofile.out" -print 2>/dev/null || true)
```

Shellcheck (0.10.0)

[warning] This action ignores everything before the -o. Use \( \) to group.

(SC2146)